src/views/Bridge.vue

```diff
@@ -466,7 +481,7 @@ export default class Bridge extends Mixins(
   }
 
   formatBalance(isSora = true): string {
-    if (!this.isRegisteredAsset || !this.asset) {
+    if (!this.asset || (!this.isRegisteredAsset && !isSora)) {
```

Comment:
`!this.asset || (!this.isRegisteredAsset && !isSora)` -> `!this.asset || !(this.isRegisteredAsset || isSora)`